misk/src/main/kotlin/misk/logging/Logging.kt

```diff
@@ -5,7 +5,7 @@ import mu.KotlinLogging
 import org.slf4j.MDC
 import org.slf4j.event.Level
 
-typealias Tag = Pair<String, String>
+typealias Tag = Pair<String, Any?>
```

Comment:
Probably shouldn't be nullable.

Reply:
Should we just skip if the tag’s value is null?

Reply:
I think seeing something like `physicalCard: null` might be helpful